hokusai/services/yaml_spec.py

```diff
@@ -1,21 +1,22 @@
 import os
+import atexit
 
 import yaml
 
-from tempfile import NamedTemporaryFile
-
-from hokusai.lib.config import config
+from hokusai.lib.config import config, HOKUSAI_TMP_DIR
 from hokusai.lib.config_loader import ConfigLoader
 from hokusai.lib.template_renderer import TemplateRenderer
 
 from hokusai.lib.exceptions import HokusaiError
 
 from hokusai.services.ecr import ECR
 
-class KubernetesSpec(object):
+class YamlSpec(object):
   def __init__(self, kubernetes_yaml):
     self.kubernetes_yaml = kubernetes_yaml
```

Comment:
Does it still make sense to name this `kubernetes_yaml`? Should it be something more generic like `yaml`?  Seems like we're generally generalizing the scope of this class, so asking the question

Reply:
Yeah so now as it would find either Kubernetes or Docker Compose templates and render them as Yaml I chose the generic?

Reply:
Not totally following your comment, but just thinking if it would make to just renaming this from `kubernetes_yaml` to `yaml` - more of a question for you than a suggestion, have pretty little context on this specific codebase.